kategory-effects/src/test/kotlin/kategory/effects/data/IOTest.kt

```diff
@@ -209,11 +209,10 @@ class IOTest : UnitSpec() {
         "IO.binding should for comprehend over IO" {
             val result = IO.binding {
                 val x = IO.pure(1).bind()
-                val y = !IO { x + 1 }
-                val z = bind { IO { y + 1 } }
-                yields(z)
+                val y = bind { IO { x + 1 } }
+                yields(y)
```

Comment:
This is because the same reason of https://github.com/kategory/kategory/pull/201#discussion_r132927084
Should I change it?